### PR TITLE
fix: small fixes

### PR DESCRIPTION
**Pull request by alexanderbira**
- Allow scrolling with open profile dropdown
- Hide admin buttons on opportunity and events table for non-admins
- Remove margin from right of viewport when modals are opened
- Don't copy from cache in Dockerfile
- Display day of the month for company users added date
- Admins don't need to accept ToS to edit student profile pages
- Update support email
- Fix MD editor/viewer paragraph and heading margins

### CODE REVIEW

File: lib/crud/students.ts [hunk 1]
```diff
@@ -20,6 +20,8 @@ const validateIsOpportunityType = (value: string | undefined): value is Opportun
 
 export const updateStudent = studentOnlyAction(
   async (_: FormPassBackState, formData: FormData, studentId: string): Promise<FormPassBackState> => {
+    const isAdmin = (await auth())!.user.role === "ADMIN"
```
Comment (IlliaDerevianko) [inline lib/crud/students.ts:23]: use `Role.ADMIN` instead

Also, add a comment explaining the not null assertion

File: app/students/[shortcode]/page.tsx [hunk 2]
```diff
@@ -154,7 +155,7 @@ const StudentProfilePage = async ({ params }: { params: { shortcode: string } })
             </Flex>
             <Box position="absolute" top="2" right="2">
               <RestrictedAreaStudent showMessage={false} studentId={studentProfile.userId}>
-                <EditStudent prevStudentProfile={studentProfile} />
+                <EditStudent prevStudentProfile={studentProfile} isAdmin={(await auth())!.user.role === "ADMIN"} />
```
Comment (IlliaDerevianko) [inline app/students/[shortcode]/page.tsx:158]: Use `Role.ADMIN`

**Review by IlliaDerevianko — CHANGES_REQUESTED** [review]

File: lib/crud/students.ts (hunk 1 above)
Comment (alexanderbira) [reply lib/crud/students.ts]: Resolved in [ab865de](https://github.com/imperial/cpp-connect/pull/88/commits/ab865de7eac50722f9c8c42f0402f3e86edc048b)

File: app/students/[shortcode]/page.tsx (hunk 2 above)
Comment (alexanderbira) [reply app/students/[shortcode]/page.tsx]: Resolved in [ab865de](https://github.com/imperial/cpp-connect/pull/88/commits/ab865de7eac50722f9c8c42f0402f3e86edc048b)

**Review by IlliaDerevianko — APPROVED** [review]

**Pull request merged**